.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -1,14 +1,29 @@
 ---
-name: Feature request
-about: Suggest an idea for this project
-
+name: 💡 Feature Request
+about: Suggest an idea for mailcow.
+labels: bug, Needs triage
```

Comment:
A feature request should not have the label bug, I would suggest using `enhancement ` or `feature` or something like this.

Reply:
Oh, yeah, for sure. Let's say this was intended and was a test to see if someone is really checking the PR. Solved - thanks!